inc/classes/optimization/CSS/class-critical-css.php

```diff
@@ -129,26 +129,19 @@ public function stop_generation() {
 	 * @author Remy Perona
 	 */
 	public function clean_critical_css() {
-		try {
-			$directory = new \RecursiveDirectoryIterator( $this->critical_css_path, \FilesystemIterator::SKIP_DOTS );
-		} catch ( \UnexpectedValueException $e ) {
-			// no logging yet.
-			return;
-		}
-
-		try {
-			$files = new \RecursiveIteratorIterator( $directory, \RecursiveIteratorIterator::CHILD_FIRST );
-		} catch ( \Exception $e ) {
-			// no logging yet.
-			return;
-		}
+		$filesystem = rocket_direct_filesystem();
+		$dir        = $filesystem->dirlist( $this->critical_css_path );
```

Comment:
`dirlist()` is not performant nor is it needed for this implementation. Instead, use SPL.